src/cli/package_manager_command.zig

```diff
@@ -270,6 +271,103 @@ pub const PackageManagerCommand = struct {
                 }
             }
 
+            Global.exit(0);
+        } else if (strings.eqlComptime(subcommand, "why")) {
+            const load_lockfile = pm.lockfile.loadFromDisk(ctx.allocator, ctx.log, "bun.lockb");
+            handleLoadLockfileErrors(load_lockfile, pm);
+
+            Output.flush();
+            Output.disableBuffering();
+            const lockfile = load_lockfile.ok.lockfile;
+
+            const param = pm.options.positionals[1];
+            const id = if (lockfile.package_index.get(String.Builder.stringHash(param))) |id| id.PackageID else {
+                Output.prettyErrorln("<r><red>error<r>: Could not find package {s}", .{param});
```

Comment:
this probably copied older code but there is a better helper for red error text

Output.errGeneric("Package \"{s}\" is not installed", .{ param });